.github/workflows/release-canary.yml

```diff
@@ -0,0 +1,33 @@
+name: release-canary
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
```suggestion
on:
  pull_request:
    types:
      - labeled
    paths:
      - 'packages/cli/src/**.ts'
```

Reply:
I didn't add the `paths` because sometimes we might want a experimental version just to test a dependency package update (`package.json` changes only). Neither, I can't test this PR because there is no TS file modified.

Reply:
`paths` is an array, and you can add the root and package `package.json` if you want. But yeah, whatever works best for you guys :+1: